Essentials/src/com/earth2me/essentials/SpawnMob.java

```diff
@@ -109,7 +109,8 @@ public static void spawnmob(final IEssentials ess, final Server server, final Co
             for (int i = 0; i < mobCount; i++) {
                 spawnMob(ess, server, sender, target, sloc, parts, data);
             }
-            sender.sendMessage(mobCount * parts.size() + " " + mob.name.toLowerCase(Locale.ENGLISH) + mob.suffix + " " + tl("spawned"));
+            // TODO: Customizing spawned message
```

Comment:
Please move this to a separate PR.

Reply:
There is only one problem. I do not know if they will show the placeholders.
mobcount etc.

Reply:
This is a separate feature to the bug fix in the PR, and it also won't work because `tl` isn't being called properly. It needs to be called with a message key first, then the placeholders to substitute in, not a concatenated message like in this PR.